### ISSUE TITLE
remouse does not start on Macos

### ISSUE THREAD

**Issue by WladyX** [Evidlo/remarkable_mouse#72]
```
Successfully installed remarkable_mouse-7.0.1
 12:18:39
❯ remouse
Connected to 10.11.99.1
Traceback (most recent call last):
  File "/opt/homebrew/bin/remouse", line 8, in <module>
    sys.exit(main())
  File "/opt/homebrew/lib/python3.9/site-packages/remarkable_mouse/remarkable_mouse.py", line 148, in main
    from remarkable_mouse.pynput import read_tablet
  File "/opt/homebrew/lib/python3.9/site-packages/remarkable_mouse/pynput.py", line 4, in <module>
    import libevdev
  File "/opt/homebrew/lib/python3.9/site-packages/libevdev/__init__.py", line 23, in <module>
    from .device import Device, InputAbsInfo
  File "/opt/homebrew/lib/python3.9/site-packages/libevdev/device.py", line 29, in <module>
    from .event import InputEvent
  File "/opt/homebrew/lib/python3.9/site-packages/libevdev/event.py", line 23, in <module>
    from .const import EventType, EventCode
  File "/opt/homebrew/lib/python3.9/site-packages/libevdev/const.py", line 427, in <module>
    _load_consts()
  File "/opt/homebrew/lib/python3.9/site-packages/libevdev/const.py", line 352, in _load_consts
    Libevdev()  # classmethods, need to make sure it's loaded at once
  File "/opt/homebrew/lib/python3.9/site-packages/libevdev/_clib.py", line 395, in __init__
    super(Libevdev, self).__init__()
  File "/opt/homebrew/lib/python3.9/site-packages/libevdev/_clib.py", line 76, in __init__
    self._load()
  File "/opt/homebrew/lib/python3.9/site-packages/libevdev/_clib.py", line 83, in _load
    cls._lib = cls._cdll()
  File "/opt/homebrew/lib/python3.9/site-packages/libevdev/_clib.py", line 123, in _cdll
    return ctypes.CDLL("libevdev.so.2", use_errno=True)
  File "/opt/homebrew/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/ctypes/__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: dlopen(libevdev.so.2, 6): image not found
```

**Comment by Evidlo** [comment]
This should be fixed in 7.0.2

**Comment by Evidlo** [comment]
This is a bug I introduced in the last release.  Try `pip install remarkable-mouse==7.0.0` or `pip install remarkable-mouse==6.0.0`
